SingularityUI/app/actions/api/base.es6

```diff
@@ -23,12 +23,12 @@ export function buildApiAction(actionName, opts = {}, keyFunc = undefined) {
     return { type: CLEAR };
   }
 
-  function started(key = undefined) {
-    return { type: STARTED, key };
+  function started(key = undefined, options = undefined) {
```

Comment:
not exactly asking you to change anything, but won't these args default to `undefined` by default? is there a benefit to being explicit here?